CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ This is not a comprehensive list of changes but rather a hand-curated collection
 
 v4.3
 ====
+- Introduced IMU component that models industrial Inertial Measurement Units (IMUs) with corresponding output signals. 
```

Comment:
- Remove "industrial"
- Could add ".. with corresponding output signals, including orientation, accelerometer, and gyroscope signals"